backend/src/main/java/com/happy/friendogly/pet/controller/PetController.java

```diff
@@ -72,4 +73,10 @@ public void update(
     ) {
         petCommandService.update(memberId, petId, request, image);
     }
+
+    @GetMapping("/exists/mine")
+    public ApiResponse<FindExistMyPetResponse> existMine(@Auth Long memberId) {
+        FindExistMyPetResponse response = petQueryService.existMine(memberId);
+        return ApiResponse.ofSuccess(response);
+    }
```

Comment:
```suggestion
    @GetMapping("/exists/mine")
    public ApiResponse<FindPetExistenceResponse> checkPetExistence(@Auth Long memberId) {
        FindPetExistenceResponse response = petQueryService.checkPetExistence(memberId);
        return ApiResponse.ofSuccess(response);
    }
```

existMine은 어색한 것 같아요. 네이밍 제안해봅니다~